src/test/java/com/rakovets/course/java/core/practice/date_and_time/DateWrapperTest.java

```diff
@@ -0,0 +1,95 @@
+package com.rakovets.course.java.core.practice.date_and_time;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.time.LocalDate;
+import java.time.temporal.Temporal;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.stream.Stream;
+
+public class DateWrapperTest {
+    @Test
+    void getDateTest() {
+        LocalDate localDate1 = DateWrapper.getLocalDate(1994, 7, 12);
+        assertEquals("1994-07-12", localDate1.toString());
+    }
+
+    @Test
+    void addMonthsTest() {
+        LocalDate localDate1 = DateWrapper.getLocalDate(1994, 7, 12);
+        LocalDate localDate2 = DateWrapper.addMonths(localDate1, 3);
+        assertEquals("1994-10-12", localDate2.toString());
+    }
+
+    @Test
+    void localDatePatternTest() {
+        LocalDate localDate1 = DateWrapper.getLocalDate(1994, 7, 12);
+        assertEquals("12 July 1994", DateWrapper.localDatePattern(localDate1, "d MMMM yyyy"));
+    }
+
+    @Test
+    void localDateFormatTest() {
+        assertEquals("2009-05-19", DateWrapper.localDateFormat("19/05/2009", "dd/MM/yyyy").toString());
+    }
+
+    @Test
+    void daysBetweenTest() {
+        LocalDate localDate1 = DateWrapper.getLocalDate(1994, 7, 12);
+        LocalDate localDate2 = DateWrapper.getLocalDate(1995, 7, 13);
+        assertEquals(366, DateWrapper.daysBetweenLocalDates(localDate1, localDate2));
+    }
+
+    @Test
+    void addDaysTest() {
+        LocalDate localDate1 = DateWrapper.getLocalDate(1994, 7, 12);
+        assertEquals("1994-07-15", localDate1.with(DateWrapper.addDays(3)).toString());
```

Comment:
"Violation: There isn't separating empty lines for GIVEN, WHEN, THEN parts
Solution: Add blank line"